WebRoot/js/ajax/dwt/widgets/DwtCheckbox.js

```diff
@@ -349,3 +378,13 @@ function(evt) {
 	checkbox.focus();
 	checkbox.notifyListeners(DwtEvent.SELECTION, selEv);
 };
+
+
+/**
+ * Set custom icon for checkbox
+ */
+DwtCheckbox.prototype.__setCustomCheckboxIcon = function () {
+	var icon = this.isSelected() ? 'CheckBoxChecked' : 'CheckBox';
+	this._inputIconEl.innerHTML = AjxImg.getImageHtml(DwtCheckbox.CUSTOM_STYLE_ICONS[icon]);
+	this._inputIconEl.setAttribute(AjxEnv.isIE ? "htmlFor" : "for", this._inputEl.id);
```

Comment:
No, not needed, I misplaced it. 
Updated code: https://github.com/Zimbra/zm-ajax/pull/4/files#diff-5d22f86ec071d8b2e4f423ae9803c185R355 Thanks for pointing it though.